packages/Compute/src/package.ts

```diff
@@ -227,3 +227,16 @@ export function modelCatalog() {
     }
   } else grok.shell.v = modelsView;
 }
+
+//name: RangeValidatorFactory
+//input: object params
+//output: object validator
+export function RangeValidatorFactory(params: any) {
```

Comment:
Suggestion: 

1) Introduce implicit first argument of type `DG.FuncCall` for all validators. Smth like `RangeValidatorFactory(currentCall:DG.FuncCall, params: any)`
2) Pass current call of the function to each validator

This way, it will be possible to depend validation result on the dynamic context. E.g. code this:
```
 return (val: number) => {
    const { min } = params;
    const { max } = currentCall.inputs['myMaxValueComputedOnPreviousStep'];
    if (val < min || val > max) {
      // TODO: format results
      return { errors: [`Out of range [${min}, ${max}] value: ${val}`] };
    }
  }
```

Reply:
Ah, I see `context` in `Validator` itself. It is OK then